.gitlab-ci.yml

```diff
@@ -9,55 +9,29 @@ stages:
 
 include:
   - project: nci-gdc/gitlab-templates
-    ref: 0.2.0
+    ref: 0.2.1
     file:
       - templates/global/full.yaml
       - templates/python/full.yaml
 
-variables:
-  BASE_CONTAINER_VERSION: "1.4.0"
-  PRE_COMMIT_DOCKER_IMAGE: docker.osdc.io/ncigdc/python3.7-builder:2.2.0
-  DOCKER_BUILDKIT: 1
-  PIP_EXTRA_INDEX_URL: https://nexus.osdc.io/repository/pypi-all/simple
-  DOCKER_BUILD_OPTS: "--build-arg PIP_EXTRA_INDEX_URL=https://nexus.osdc.io/repository/pypi-all/simple"
-  DOCKER_PUSH_OPTS: "--all-tags"
-  TWINE_REPOSITORY_URL: "https://nexus.osdc.io/repository/pypi-snapshots/"
+pre-commit:
+  before_script:
+    - export
```

Comment:
One of the things that I am not enjoying about these files (that is no fault of your own) is the terrible locality of the information. _What does export do? Export files? Export environment variables? This is going to run 'export' but where does export come from? Probably a template... which template global/full or python/full.. I need to go search._

Reply:
export will print out environment variables help us to debug gitlab-ci. 
https://docs.gitlab.com/ee/ci/variables/#list-all-variables

I put export in pre-commit because it usually exists in every pipeline and has less output than other jobs.

Reply:
I will add a comment about this